packages/openops/src/lib/openops-tables/rows.ts

```diff
@@ -63,96 +63,147 @@ const semaphore = TablesAccessSemaphore.getInstance();
-    }
+  params.append('user_field_names', `true`);
+  getRowsParams.filters?.forEach((filter) => {
+    params.append(
+      `${buildSimpleFilterUrlParam(`${filter.fieldName}`, filter.type)}`,
+      `${filter.value}`,
+    );
+  });
+  if (getRowsParams.filterType) {
+    params.append('filter_type', `${getRowsParams.filterType}`);
+  }
 
-    const paramsString = params.toString();
-    const baseUrl = `api/database/rows/table/${getRowsParams.tableId}/`;
-    const url = paramsString ? baseUrl + `?${paramsString}` : baseUrl;
+  const paramsString = params.toString();
+  const baseUrl = `api/database/rows/table/${getRowsParams.tableId}/`;
+  const url = paramsString ? baseUrl + `?${paramsString}` : baseUrl;
+  const authenticationHeader = createAxiosHeaders(getRowsParams.token);
 
-    const authenticationHeader = createAxiosHeaders(getRowsParams.token);
-    const getRowsResult = await makeOpenOpsTablesGet<{ results: any[] }[]>(
-      url,
-      authenticationHeader,
-    );
+  return executeWithConcurrencyLimit(
+    async () => {
+      const getRowsResult = await makeOpenOpsTablesGet<{ results: any[] }[]>(
+        url,
+        authenticationHeader,
+      );
 
-    return getRowsResult.flatMap((row: any) => row.results);
-  });
+      return getRowsResult.flatMap((row: any) => row.results);
+    },
+    (error) => {
+      logger.error('Error while getting rows:', {
+        error,
+        url,
+        ...getRowsParams,
+      });
+    },
+  );
 }
 
 export async function updateRow(updateRowParams: UpdateRowParams) {
-  return executeWithConcurrencyLimit(async () => {
-    const authenticationHeader = createAxiosHeaders(updateRowParams.token);
-    return await makeOpenOpsTablesPatch(
-      `api/database/rows/table/${updateRowParams.tableId}/${updateRowParams.rowId}/?user_field_names=true`,
-      updateRowParams.fields,
-      authenticationHeader,
-    );
-  });
+  const url = `api/database/rows/table/${updateRowParams.tableId}/${updateRowParams.rowId}/?user_field_names=true`;
+
+  return executeWithConcurrencyLimit(
+    async () => {
+      const authenticationHeader = createAxiosHeaders(updateRowParams.token);
+      return await makeOpenOpsTablesPatch(
+        url,
+        updateRowParams.fields,
+        authenticationHeader,
+      );
+    },
+    (error) => {
+      logger.error('Error while updating row:', {
+        error,
+        url,
+        ...updateRowParams,
+      });
+    },
+  );
 }
 
 export async function upsertRow(upsertRowParams: UpsertRowParams) {
-  return executeWithConcurrencyLimit(async () => {
-    const authenticationHeader = createAxiosHeaders(upsertRowParams.token);
-    return await makeOpenOpsTablesPut(
-      `api/database/rows/table/${upsertRowParams.tableId}/upsert/?user_field_names=true`,
-      upsertRowParams.fields,
-      authenticationHeader,
-    );
-  });
+  const url = `api/database/rows/table/${upsertRowParams.tableId}/upsert/?user_field_names=true`;
```

Comment:
took it outside of executeWithConcurrencyLimit, so that I will have the URL for the logs, and it doesn't need to be inside the lock anyway